app/assets/javascripts/scp-related-genes-ideogram.js

```diff
@@ -103,6 +102,35 @@ function onPlotRelatedGenes() {
   window.SCP.log('ideogram:related-genes', props)
 }
 
+/**
+ * Set inner visibility of ideogram
+ *
+ * @param found Boolean Whether related genes have been found
+ */
+function setFoundRelatedGenes(found) {
```

Comment:
Bootstrap has the built-in class 'hidden' that should simplify this and not require new css.  It looks like you want to start with the wrapper having the 'hidden' class, and then on finding related changes, toggle the 'hidden' class appropriately

Reply:
Good to know!  Done in 9e52773.